docs/KustoSource.md

```diff
@@ -21,7 +21,7 @@ that is using it. Please verify the following before using Kusto connector:
  
  Kusto connector implements Spark 'Datasource V1' API. 
  Kusto data source identifier is "com.microsoft.kusto.spark.datasource". 
- Kusto table schema is translated into a spark as explained in [DataTypes](Spark-Kusto DataTypes mapping.md).
+ Kusto table schema is translated into a spark as explained in [Spark-Kusto DataTypes](mapping.md).
```

Comment:
>  into a spark 
what does this mean?

Reply:
nothing

Reply:
hmm was it supposed to say `into a spark scheme`?